src/cascadia/TerminalSettingsEditor/EditColorScheme.xaml

```diff
@@ -450,6 +450,39 @@
                     </Button>
                 </StackPanel>
             </local:SettingContainer>
+
+            <local:SettingContainer x:Name="SetAsDefaultContainer"
+                                    x:Uid="ColorScheme_SetAsDefault">
+                <Button x:Uid="ColorScheme_SetAsDefaultButton"
+                        Style="{StaticResource BrowseButtonStyle}"
+                        Click="{x:Bind ViewModel.SetAsDefault_Click}"/>
+            </local:SettingContainer>
+
+            <Border MaxWidth="{StaticResource StandardControlMaxWidth}">
+                <Button x:Name="DeleteButton"

```

Comment:
**Design opinion**: instead of using visibility, we should enable/disable the button and display the disclaimer next to it when it's disabled.

Reply:
The `RenameContainer` already contains the disclaimer that this is an in box scheme and cannot be renamed/deleted, so I opted to just not show the delete button at all for a cleaner look. Happy to be overridden if the consensus is to show the delete button

Reply:
Polling for opinions: @zadjii-msft @DHowett @lhecker